usr/src/lib/libdtrace/common/dt_parser.h

```diff
@@ -27,6 +27,10 @@
  * Copyright (c) 2013 Joyent, Inc. All rights reserved.
  */
 
+/*
+ * Copyright (c) 2014 by Delphix. All rights reserved.
```

Comment:
I've fixed this, and looked for similar errors in the other modified files.  They all looked OK.